docs/repositories/issues.md

```diff
@@ -55,7 +57,7 @@ You can define one or more of the following filters:
 
 Use the options in the menu of each issue to:
 
--   **Copy the link** to the issue.
+-   **Copy the link** to the issue (available only for issues listed on the Issues page).<!-- TODO ALA-643 confirm -->
```

Comment:
I'm not seeing the "copy link" menu entry on commit/PR issues. I guess we can't link directly, so we removed it?

Reply:
we only show that option if this condition is met: issue.resultDataId && issueType === 'current'
![Screenshot 2024-03-01 at 17 11 34](https://github.com/codacy/docs/assets/28684353/959917d1-1950-46c4-99c9-de21b2411ccc)

since that issue is still not on the main branch, it doesnt have a type yet(?)

@alerizzo can you confirm this?